### DISCUSSION TITLE
How to hide the (1) from the navigation bar when using annotations in a header?

### DISCUSSION THREAD

**Discussion by laikrai** [squidfunk/mkdocs-material#6693] [Q&A] [answered]
![image](https://github.com/squidfunk/mkdocs-material/assets/71832599/ba87745c-0ab6-498d-a0fc-760b327cce03)

**Comment by squidfunk** [comment, marked as answer]
You can pass an explicit table of contents label, if [Attribute Lists](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists) is enabled:

``` md
## Join (1) { data-toc-label="Join" }
```

Other than that, using annotations in headlines might not be the best idea. We've just added [footnote tooltips](https://squidfunk.github.io/mkdocs-material/reference/footnotes/#footnote-tooltips), which have better Markdown support than annotations as they degrade more gracefully.

**Reply by kamilkrzyskow** [reply]
The screenshot shows the integrated table of contents, so the `toc-item.html` partial is responsible for rendering it. 
Additionally to the above the `(1)` can be removed via some context filter (jinja should have access to `str` functions) in a template override:
https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme
https://github.com/squidfunk/mkdocs-material/blob/f55ce2c81b6c391c40a139660e3a1e0032182848/src/templates/partials/toc-item.html#L27
Otherwise the `page.toc` can be modified by an MkDocs hook in the `on_page_context` event to remove the `(1)` from the title:
https://www.mkdocs.org/user-guide/configuration/#hooks
https://www.mkdocs.org/dev-guide/plugins/#on_page_context
